lib/pulsar.ex

```diff
@@ -131,6 +131,8 @@ defmodule Pulsar do
           clients
       end
 
+    Pulsar.ProducerEpochStore.init()
```

Comment:
I am thinking that it may be best to move this to the client? The same applies to `Pulsar.ProducerEpochStore.cleanup()`. The idea is for clients to be fully isolated from one another. What do you think?

Reply:
I changed the ets table to be client specific and removed the global clean up altogether. The client would be the process owner of the table, so when it shuts down the table would automatically be deleted by the beam.